docs/source/joining_datasets.md

```diff
@@ -43,7 +43,7 @@ splits:
                         subflavors: # If needed, will be merged(overriding) with parent subflavor
                             ds2_extra: 2
                         split_config: split.yaml  # Sets this for all joined datasets
-                join_type:
+                joiner:
                     __module__: my_module
                     __class__: JoinedSample # Type should implement from_joined(ds1, ds2)
```

Comment:
I believe this is still wrong. Should be the joiner method, no?

Reply:
See below